src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -79,4 +82,41 @@ public void maybeChangeIsScottish(RegionalProcessingCenter oldRpc, RegionalProce
             caseData.setIsScottishCase(isScottishCase);
         }
     }
+
+    private void checkAddresses(SscsCaseData sscsCaseData, PreSubmitCallbackResponse response) {
```

Comment:
we should avoid returning void if we are changing data. i.e. return the updated response.

Also "checkAddresses" is a little ambigous method name. Can you think of anything more descriptive?